starter-code/main.js

```diff
@@ -53,6 +53,217 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
             })
             break;
 
+          case "4":
+            // 4.- List by name all companies founded in february of 2004.?
+            db.collection('companies').find({founded_year : 2004}, { name: 1, _id: 0 }).toArray((error, result) => {
```

Comment:
Fundadas en febrero!
`.find({ $and: [{ founded_year: 2004 }, { founded_month: 2 }] }, { name: 1, _id: 0 })`

Reply:
Gracias!
De nuevo, no leí hasta el final.